### PR TITLE
add missing area parameter for eta_m arg

### LINKED ISSUE #1151: pvsyst cell temp docs for eta_m arg missing "area" from definition
**Describe the bug**
the definition for `eta_m` says:
image

But I think this should actually be: `(total DC power) / (poa_global * (total active module area))`

**To Reproduce**
1. Go to https://pvlib-python.readthedocs.io/en/latest/generated/pvlib.temperature.pvsyst_cell.html#pvlib.temperature.pvsyst_cell
2. Look at `pvlib.temperature.pvsyst_cell` parameters
3. Scroll down to `eta_m`
4. See definition

**Expected behavior**
I think this should actually be: `(total DC power) / (poa_global * (total active module area))`

**Screenshots**
image

**Versions:**
 - ``pvlib.__version__``: v0.8.1
 - ``pandas.__version__``: na
 - python: na

**Additional context**
see https://www.pvsyst.com/help/thermal_loss.htm
>`Effic`  is the PV efficiency (related to the module area), i.e. the energy removed from the module. When possible, the PV efficiency is calculated according to the operating conditions of the module. Otherwise it is taken as 10%.

Does this mean that the `Uc` and `Uv` coefficients already take into account the active module area? but then `eta_m` will not be a number less than one, so I don't think so.

### PR DESCRIPTION
**Pull request by KushalBeniwal**
- [x] Closes #1151
 - [x] I am familiar with the [contributing guidelines](https://pvlib-python.readthedocs.io/en/latest/contributing.html)
 - [x] Pull request is nearly complete and ready for detailed review.

Before:
Screenshot (1)

After:
Screenshot (2)

### CODE REVIEW

**Comment by mikofski** [comment]
Hi Kushal, can you check stickler? also what do you think about formatting the math using latex so it looks better? Thnaks!

**Comment by KushalBeniwal** [comment]
Screenshot (6)
This is what it looks like now.

File: pvlib/temperature.py [hunk 1]
```diff
@@ -316,7 +316,9 @@ def pvsyst_cell(poa_global, temp_air, wind_speed=1.0, u_c=29.0, u_v=0.0,
         :math:`\left[ \frac{\text{W}/\text{m}^2}{\text{C}\ \left( \text{m/s} \right)} \right]`
 
     eta_m : numeric, default 0.1
-        Module external efficiency as a fraction, i.e., DC power / poa_global.
+        Module external efficiency as a fraction, i.e.,
+        :math:`(total\ DC\ power)/(poa\_global \times
+        (number\ of\ cells \times cell\ area))`.
```
[the remark this answers is not in the document]
Comment (KushalBeniwal) [reply pvlib/temperature.py:321]: Thanks @cwhanse for your suggestion, I'll commit the suggested changes.

[the remark this answers is not in the document]
Comment (mikofski) [reply pvlib/temperature.py]: Sorry, @kushalBeniwal it needs to be "total module area" as in the area of all of the PV panels in the entire field.

**Comment by mikofski** [comment]
@KushalBeniwal Sorry, I think we should remove the underscores. Also what would it look like if we use `\frac {} {}`?  EG: something like this:
external efficiency
or more tersely:
external efficiency tersely
where `SUM(P_dc)` is the total DC power, `POA_global` is the global irradiance in the plane of the array, and `SUM(A_module)` is the total module surface area.

@cwhanse  any preference?

**Comment by cwhanse** [comment]
> @cwhanse any preference?

@mikofski you aren't wrong - the DC power and area need to match. But I think it's confusing, and out of synch with the rest of pvlib, to have this function ask for a parameter based on array scale values, rather than single module values. E.g. all the methods in `PVSystem` and `ModelChain` calculate DC power for a single module. I think the cell temperature functions should do likewise.

I don't being formal with the equation adds value here. One can't use the output of the DC model as input to this temperature function (it's circular) so I wouldn't use pvlib variable names to avoid suggesting that's what is needed.

I guess I'd prefer the text formal "DC power / (POA irradiance x module area)". OK with me if you'd prefer to render that using LateX, as "P_{DC} / (G_{POA} \times Area}" and then defining each of the three terms.

Sorry @KushalBeniwal that we're having a bit of back and forth on this.

**Comment by mikofski** [comment]
Agreed.  Let's stop the word "total" keep it away the module level

**Comment by KushalBeniwal** [comment]
> Sorry @KushalBeniwal that we're having a bit of back and forth on this.

Not an issue.
I should have first asked for your opinion before making any changes.

**Comment by KushalBeniwal** [comment]
I tried both, so, which one looks better or should I try something else?
1. Screenshot (7)
2. Screenshot (9)
Is "Area" fine or should I put {A}_module?

**Comment by mikofski** [comment]
My vote's on 1.

**Comment by cwhanse** [comment]
> My vote's on 1.

Mine also.

**Review by cwhanse — APPROVED** [review]

**Comment by mikofski** [comment]
Thanks @KushalBeniwal !

**Pull request merged**